library/testing/.clang-format

```diff
@@ -0,0 +1,17 @@
+BasedOnStyle: InheritParentConfig
+BreakBeforeBraces: Custom
+BraceWrapping:
+  # emulate parent config's Allman...
+  AfterClass: true
+  AfterControlStatement: true
+  AfterEnum: true
+  AfterFunction: true
+  AfterNamespace: true
+  AfterObjCDeclaration: true
+  AfterStruct: true
+  AfterUnion: true
+  BeforeCatch: true
+  BeforeElse: true
+  IndentBraces: false
+  # ...except for lambdas
+  BeforeLambdaBody: false
```

Comment:
why does this matter ?

Reply:
the idea is to keep the global project style except for lambdas that get real noisy real fast, see https://github.com/f3d-app/f3d/pull/1585#discussion_r1739634772

Reply:
thanks for the reminder